sdk/react/bottomOverlay.js

```diff
@@ -205,14 +193,14 @@ const BottomOverlay = createReactClass({
             ref='progressBar'
             percent={percent}
             config={this.props.config}
-            ad={this.props.ad}
+            ad={this.props.ad != null}
```

Comment:
this is not the same

Reply:
I've changed logic since this property is not used as "property". It's only being checked for presence
see `adBar.js`

Reply:
so just make it boolean: `ad={!!this.props.ad}`
not strict comparison with `null` looks confusing